libs/platform/src/lib/components/object-attribute/object-attribute.component.ts

```diff
@@ -0,0 +1,55 @@
+import {
+    Component, Input, ChangeDetectionStrategy, Output,
+    EventEmitter, OnInit, ElementRef, Renderer2, ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+    selector: 'fdp-object-attribute',
+    templateUrl: './object-attribute.component.html',
+    styleUrls: ['./object-attribute.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ObjectAttributeComponent implements OnInit {
+
+    /** label for the element */
+    @Input()
+    label: string;
+
+    /** Text in link */
+    @Input()
+    linkText: string;
+
+    /** Is linked object attribute */
+    @Input()
+    islink: boolean;
+
+    /** To disabled linked object attribute */
+    @Input()
+    disabled: boolean;
+
+    /** Emitting object attribute link click event */
+    @Output()
+    objectAttributeclick: EventEmitter<MouseEvent | KeyboardEvent | TouchEvent> = new EventEmitter();
```

Comment:
There is no added value listing types like this. Why dont you use just Event if you want to have some type ?